spec/Tagger/Repository/Content/InMemory.hs

```diff
@@ -1,4 +1,4 @@
-module InMemoryContentRepository where
+module Tagger.Repository.Content.InMemory (inMemoryContentRepository) where
```

Comment:
I'm not sure I understand the rationale beyond this move.

`Tagger` is the namespace reserved for the domain of the application.
Implementation of repositories are not part of the domain, so they do not belong there.
I would understand a move to `Infrastructure/Persistence/InMemoryContentRepository.hs`.

Moreover, why move `Content` to the namespace? It is not coherent with the rest of the structure of the project

Reply:
The idea was to have an abstract representation of the repository and then specific concrete implementations:

- `Tagger.Repository.Content` (the abstract repository)
- `Tagger.Repository.Content.InMemory` (concrete implementation)
- `Tagger.Repository.Content.Postgres`(concrete implementation)

But maybe it makes more sense to put it in the Application layer?

- `Api.Repository.Content.InMemory`
- `Api.Repository.Content.Postgres`

Reply:
concrete implementations are part of the infrastructure